cmstestsuite/unit_tests/grading/steps/utils_test.py

```diff
@@ -122,7 +122,7 @@ def test_multiple_commands_success(self):
                              collect_output=True)
 
         # 2 commands executed, with exec_num 0 and 1
-        self.assertEquals(self.sandbox.exec_num, 1)
+        self.assertEqual(self.sandbox.exec_num, 1)
```

Comment:
Optional: consider moving the normalization of assertEquals to a different commit.

Reply:
Moved to `821e1b2b474d885f910f519101d06f6b97133576`